net/pkt/pkt_send.c

```diff
@@ -162,7 +162,7 @@ ssize_t psock_pkt_send(FAR struct socket *psock, FAR const void *buf,
 
   /* Verify that the sockfd corresponds to valid, allocated socket */
 
-  if (!psock || psock->s_crefs <= 0)
+  if (!psock || !psock->s_conn)
```

Comment:
Throughout the PR, `psock->s_conn` is both being check against NULL (which I prefer), and used as input to operator `!`.
As a suggestion, it is better to always compare it with NULL, since `s_conn` is in fact a pointer.

Reply:
Done!